crowdsourcing/crowdsourcing_tutorial.py

```diff
@@ -290,3 +294,11 @@ def encode_text(text):
 sklearn_model.fit(train_vectors, probs_to_preds(Y_train_prob))
 
 print(f"Accuracy of trained model: {sklearn_model.score(test_vectors, Y_test)}")
+
```

Comment:
small comment about accuracy. almost as good as having crowd labels in perpetuity for all unseen examples

Reply:
Will add. I didn't originally do this because 86% vs 92% is a pretty big difference, so I don't want it to look like we're over-claiming.

Reply:
it's pretty good given that we have no crowdsourcing labels for test! definitely acknowledge that it's not perfect